message_ix/model/MESSAGE/parameter_def.gms

```diff
@@ -229,6 +229,16 @@ Parameters
 * technology input-output mapping and costs parameters
     input(node,tec,vintage,year_all,mode,node,commodity,level,time,time)  relative share of input per unit of activity
     output(node,tec,vintage,year_all,mode,node,commodity,level,time,time) relative share of output per unit of activity
+* +++++
+* variant 1: commodity input and output associated with construction and retirement of capacity
+    input_cap_new(node,tec,vintage,node,commodity,level,time)  relative share of input per unit of new capacity built
+    output_cap_new(node,tec,vintage,node,commodity,level,time) relative share of output per unit of new capacity built
```

Comment:
I can understand `input_cap_new` will specify commodity input flow for certain capacity construction, but I am not entirely sure how `output_cap_new` is relevant to a commodity flow. I can see we can still keep the `output_cap_new` part for consistency of the formulation, but practically isn't it simpler without the part?

Reply:
out_cap_new could be used for specifying generation of new scrap upon construction and more broadly I think we want to develop a formulation that allows going beyond a narrow use case that we may have at hand. Based on the comments in the modeling meeting yesterday, I tend towards going with variant 2 which is more flexible as it would also allow adding commodity requirements for maintainig the capacity (e.g., materials for rennovation of a building after a given amount of time).